src/app/code/community/Shopgate/Cloudapi/etc/config.xml

```diff
@@ -167,6 +167,8 @@
                         .page-title,
                         .customer-account-create .rewards,
                         .customer-account-create .back-link,
+                        .customer-account-forgotpassword .back-link,
+                        #login-form,
```

Comment:
should this have a page handle of some sort? I worry about these open css display non's ^_^

Reply:
extended to feel save ^^